AirCasting/CreateSessionViews/ConnectingABView.swift

```diff
@@ -5,18 +5,18 @@
 //  Created by Lunar on 04/02/2021.
 //
 
-import SwiftUI
 import CoreBluetooth
+import SwiftUI
 
 struct ConnectingABView: View {
-    
+    @Environment(\.presentationMode) var presentationMode
```

Comment:
A view should not manage its own dismissal or presentation. Let's leave that in here, but in the future let's handle such stuff from a parent view.